build.gradle.kts

```diff
@@ -2,3 +2,15 @@
 plugins {
     id(backbase.plugins.dependency.updates.get().pluginId)
 }
+subprojects{
```

Comment:
Is this fix intended to be published alongside the other changes?

Reply:
Yes, I had an issue were multiple versions of the same dependency being found  in Android Studio project view.